stress/common/src/main/java/alluxio/stress/worker/WorkerBenchParameters.java

```diff
@@ -86,6 +86,14 @@ public final class WorkerBenchParameters extends FileSystemParameters {
       description = "The random max length upper bound")
   public String mRandomMinReadLength = "1m";
 
+  @Parameter(names = {"--random-min-buffer-size"},
+      description = "The buffer size for random IO operations. (1k, 16k, etc.)")
```

Comment:
Can you update this description so that we can exactly know this parameter stand for `min` random buffer size.

Reply:
@maobaolong done